sdk/state/state.go

```diff
@@ -86,6 +87,14 @@ func (c *Channel) LatestCloseAgreement() CloseAgreement {
 	return c.latestAuthorizedCloseAgreement
 }
 
+func (c *Channel) UpdateLocalEscrowAccountBalance(balance int64) {
```

Comment:
💭 `UpdateRemoteEscrowAccountBalance` is currently only used in the tests, so this is dependent on the orchestration code to do the updating? I'm wondering
1. should we be updating the balance w/in the state machine? Eg. in ConfirmPayment
2. if not 1. Then should this be living on the state machine since? ie should we make it a method in the test, or if you foresee the orchestration code using then makes sense to me keep it here

Reply:
I don't think the state machine should update itself because that requires a call out to Horizon. The application or code that uses the state machine needs to take care of knowing how to call out to Horizon (or tracking balances some other way).

> should we make it a method in the test

I don't think I understand this suggestion. Would the test update the internal value of the escrow account balance? I think we need some way for the caller to update the balance since the state machine can't do it on its own.

Reply:
> I don't think the state machine should update itself because that requires a call out to Horizon
true agreed, for a moment I forgot about the reason for this PR 😆 

>Would the test update the internal value of the escrow account balance?
true, disregard

sgtm to leave as is thx 👍